pkg/manager/diff.go

```diff
@@ -45,9 +45,15 @@ func RunDiffFuzzer(ctx context.Context, baseCfg, newCfg *mgrconfig.Config, debug
 	if err != nil {
 		return err
 	}
-	go func() {
-		new.candidates <- LoadSeeds(newCfg, true).Candidates
-	}()
+	eg, ctx := errgroup.WithContext(ctx)
+	eg.Go(func() error {
+		info, err := LoadSeeds(newCfg, true)
+		if err != nil {
+			return nil
+		}
+		new.candidates <- info.Candidates
```

Comment:
In the case of error we don't send to new.candidates, will diffCtx.Loop return in such case? Otherwise eg.Wait() below will hang.

Reply:
Added a `select` to the place where we read from `new.candidates`